src/components/TextInput/BaseTextInput/index.native.js

```diff
@@ -172,10 +173,12 @@ function BaseTextInput(props) {
     /**
      * Set Value & activateLabel
      *
-     * @param {String} value
+     * @param {String} val
```

Comment:
@tienifr i think we should revert this change, from my quick search it looks like we use `value` in many places, and almost never use `val` as a param name. Can you revert or explain why you think this is better please?

```suggestion
     * @param {String} value
```